### PR TITLE
Feature: add more test cases for SearchPublicAddressCommandParserTest

### LINKED ISSUE #90: Fix failing Github actions checks (Style and test)

### PR DESCRIPTION
**Pull request by travisim**
Fixes #90. Added more test cases for parser.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/SearchPublicAddressCommandParser.java [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
+        // if more than 1 pa entered
+        if (argMultimap.getAllValues(PREFIX_PUBLIC_ADDRESS).size() > 1) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    SearchPublicAddressCommand.MESSAGE_USAGE));
+        }
+        //if no pa entered
         if (argMultimap.getValue(PREFIX_PUBLIC_ADDRESS).isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                     SearchPublicAddressCommand.MESSAGE_USAGE));
         }
```
Comment (TheRareFox) [inline src/main/java/seedu/address/logic/parser/SearchPublicAddressCommandParser.java:33]: These two can be combined together with an or statement

File: src/main/java/seedu/address/logic/commands/SearchPublicAddressCommand.java [hunk 2]
```diff
@@ -13,12 +14,12 @@ public class SearchPublicAddressCommand extends Command {
...
     public static final String MESSAGE_USAGE = COMMAND_WORD
-        + ": Searches for a public address and returns the user, network and tag "
+            + ": Searches for a public address and returns the user, network and tag "
             + "throughout all the public addresses of all networks in the address book.\n"
-            + "Parameters: PUBLIC_ADDRESS (must be a string) "
-            + PREFIX_PUBLIC_ADDRESS + "[PUBLIC_ADDRESS]\n"
-            + "Example: " + COMMAND_WORD + " " + PREFIX_PUBLIC_ADDRESS
-            + "0x28f91d6e72eaf4372892e6c6e45dc41b574163e9fcdf94f4997958b46d772fa2";
+            + "Parameters: PUBLIC_ADDRESS (must be a string) " + PREFIX_PUBLIC_ADDRESS
+            + "[PUBLIC_ADDRESS]\n"
```
Comment (TheRareFox) [inline src/main/java/seedu/address/logic/commands/SearchPublicAddressCommand.java:20]: Square brackets are reserved for optional parameters, should instead be PREFIX + PUBLIC ADDRESS.

**Review by TheRareFox — APPROVED** [review]
Overall, LGTM. These small comments can be rectified in another commit

File: src/main/java/seedu/address/logic/commands/SearchPublicAddressCommand.java (hunk 2 above)
Comment (travisim) [reply src/main/java/seedu/address/logic/commands/SearchPublicAddressCommand.java:20]: Thx sir, very detailed, very precise.

**Pull request merged**